app/models/database_seed.rb

```diff
@@ -0,0 +1,2 @@
+class DatabaseSeed < ApplicationRecord
```

Comment:
I still feel it's weird to have a "singleton table". I would prefer something more generic such as `ForemanInternal` or some such that is a key value store that could be used for other internal details users shouldn't care much about. I suppose if we go over all settings we would probably find a few others that could be moved to it and were only there because we have no other good place to store them persistently (and maybe also things that are currently hard coded or saved in various config yamls).